target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dts

```diff
@@ -399,6 +399,18 @@
 	};
 };
 
+&factory {
+	compatible = "nvmem-cells";
+
+	macaddr_gmac1: macaddr_gmac1@0 {
```

Comment:
Solely cosmetic, but do the nodes need unique names?

Since we have DT labels in any case, maybe something like this would be enough:

`macaddr_gmac1: macaddr@0 {`

?

Reply:
I need to check if the nvmem framework like that.

Reply:
Note that this is only an open suggestion, since I do not really think the duplication is necessary here.

Reply:
no problem for nvmem so i will use the generic macaddr@....